app/routes/goal_routes.py

```diff
@@ -0,0 +1,90 @@
+from flask import Blueprint, request, jsonify, make_response, abort
+from app.models.goal import Goal
+from app.models.task import Task
+from .routes_helpers import validate_model, delete_message
+from app import db
+
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# CREATE GOAL ENDPOINT
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+
+    if not request_body.get("title"):
+        abort(make_response({"details": "Invalid data"}, 400))
+    else:
+        new_goal = Goal.from_dict(request_body)
```

Comment:
Since you `abort` the request in the `if` block, no need to put this behind an `else`